lib/factor/factorQuadratic.js

```diff
@@ -56,35 +69,13 @@ function factorQuadratic(node) {
     cValue = -cValue;
```

Comment:
these should default to 0 above, or else negating doesn't really make sense in the null case